### PR TITLE
Add a new Unconfusable Latin table

### PR DESCRIPTION
**Pull request by CydeWeys**
This new table has just been approved by ICANN. It is the same as our existing Extended Latin table, except with the removal of some lesser-used characters with diacritic marks that are confusable variants.

Thanks to Kalina for generating and getting approval for this table.

- - -
This change is [Reviewable](https://reviewable.io/reviews/google/nomulus/1981)

### CODE REVIEW

**Comment by CydeWeys** [comment]
@kokosta -- see this

**Review by kokosta — COMMENTED** [review]
LGTM

> Reviewed 5 of 6 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981#-:-NSHpqH69ogigssAxWYQ:bb6ywmx)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @gbrodman)

File: core/src/main/java/google/registry/idn/Latin-IDN.txt [hunk 1]
```diff
@@ -68,6 +68,7 @@ U+011F                         # LATIN SMALL LETTER G WITH BREVE
 U+01E7                         # LATIN SMALL LETTER G WITH CARON
 U+0121                         # LATIN SMALL LETTER G WITH DOT ABOVE
 U+0123                         # LATIN SMALL LETTER G WITH CEDILLA
+U+01E5                         # LATIN SMALL LETTER G WITH STROKE
```
Comment (gbrodman) [inline core/src/main/java/google/registry/idn/Latin-IDN.txt:71]: where does this come from?

**Review by CydeWeys — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981)* status: all files reviewed, 1 unresolved discussion (waiting on @gbrodman)

___
*[`core/src/main/java/google/registry/idn/Latin-IDN.txt` line 71 at r2](https://reviewable.io/reviews/google/nomulus/1981#-NSHfRdA3ICawaW88AnM-r2-71:-NSHwHpb8WhHQWLNRebl:b-j2j7ly) ([raw file](https://github.com/google/nomulus/blob/b1c662e589abd9f455932cc2e46131a9cb617ec2/core/src/main/java/google/registry/idn/Latin-IDN.txt#L71)):*
<details><summary><i>Previously, gbrodman wrote…</i></summary><blockquote>

where does this come from?
</blockquote></details>

See the commit description.

File: core/src/main/java/google/registry/idn/Latin-IDN.txt (hunk 1 above)
Comment (gbrodman) [reply core/src/main/java/google/registry/idn/Latin-IDN.txt:71]: nvm kalina explained

**Review by gbrodman — APPROVED** [review]

**Review by gbrodman — APPROVED** [review]
> Reviewed 5 of 6 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @CydeWeys)

___
*[`core/src/main/java/google/registry/idn/Latin-IDN.txt` line 71 at r2](https://reviewable.io/reviews/google/nomulus/1981#-NSHfRdA3ICawaW88AnM-r2-71:-NSI-ROb2xRWMAkOeU5c:bzgrof6) ([raw file](https://github.com/google/nomulus/blob/b1c662e589abd9f455932cc2e46131a9cb617ec2/core/src/main/java/google/registry/idn/Latin-IDN.txt#L71)):*
<details><summary><i>Previously, CydeWeys (Ben McIlwain) wrote…</i></summary><blockquote>

Your comment is still blocking.
</blockquote></details>

commit description did not explain it but kalina did, we're all good

**Review by CydeWeys — COMMENTED** [review]
PTAL, made some changes.

> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981#-:-NSIIdLO51JU_rXgaR4d:bgrgvdg)* status: 4 of 6 files reviewed, all discussions resolved (waiting on @gbrodman and @kokosta)

**Review by gbrodman — APPROVED** [review]
> Reviewed 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @CydeWeys)

**Review by CydeWeys — COMMENTED** [review]
PTAL again, had to change the table to make it pass validation (needed to add the `Policy:` and `URL:` fields).

> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981#-:-NSMMgJ6ENrTSGl1NSJB:b-7bby0x)* status: 4 of 6 files reviewed, all discussions resolved (waiting on @gbrodman and @kokosta)

**Review by gbrodman — APPROVED** [review]
> Reviewed 2 of 2 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @CydeWeys)

**Review by CydeWeys — COMMENTED** [review]
PTAL yet again, had to make RDE changes as well.

> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981#-:-NSMavRU8_8CuGgjSyIw:byfu5zi)* status: 6 of 11 files reviewed, all discussions resolved (waiting on @gbrodman and @kokosta)

**Review by gbrodman — APPROVED** [review]
> Reviewed 5 of 5 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1981)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @CydeWeys)

**Pull request merged**